lib/beaker/tasks/quick_start.rb

```diff
@@ -0,0 +1,49 @@
+require 'beaker-hostgenerator'
+
+namespace :beaker_quickstart do
+
+  desc 'Generate Default Beaker Host Config File'
+  task :gen_hosts do
```

Comment:
The secondary level of namespaces would be nice to have, considering that there will be variants on some of the generation tasks in the follow-up work. But this is a nice-to-have, so it's not PR merge blocking.